src/context.rs

```diff
@@ -148,6 +145,14 @@ impl Context {
         }
     }
 
+    pub(crate) fn version_range(&self) -> Option<VersionRange> {
+        self.args.version_range
+    }
+
+    pub(crate) fn version_step(&self) -> u16 {
+        self.args.version_step
+    }
+
```

Comment:
I believe these fields are already accessible via `impl Deref for Context`.

Reply:
Missed that, thanks